Meta.svelte

```diff
@@ -3,27 +3,133 @@
   Metadata helper
 -->
 <script>
-  /** Page title */
+
+  interface TwitterCardProps {
+    card: Optional<string>,
+    site: Optional<string>,
+    creator: Optional<string>,
+  }
+
+  interface OpenGraphProps {
+    type: Optional<string>,
+    locale: Optional<string>,
+    siteName: Optional<string>,
+  }
+
+  interface ImageProps {
+    url: Optional<string>,
+    width: Optional<string>,
+    height: Optional<string>,
+    alt: Optional<string>,
+  }
+
+  /** Page title **/
   export let title: Optional<string> = undefined;
-  /** Page description */
+  /** Page description **/
   export let description: Optional<string> = undefined;
-  /** Page image */
-  export let image: Optional<string> = undefined;
+  /** Page image **/
+  export let image: ImageProps = {
+    url: undefined,
+    width: undefined,
+    height: undefined,
+    alt: title || undefined,
+  };
+  /** Page url **/
+  export let url: Optional<string> = undefined;
+  /** Site logo **/
+  export let logoUrl: Optional<string> = undefined;
+  /** Search URL **/
+  export let searchUrl: Optional<string> = undefined;
+  /** Sitemap URL **/
+  export let sitemapUrl: Optional<string> = undefined;
+  /** Twitter Options **/
+  export let twitter: TwitterCardProps = {
```

Comment:
The defaults don't match the interface, should prob add all possible types to the interface (also it can be a type, since it doesn't need to be extended for etc)